packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css

```diff
@@ -0,0 +1,312 @@
+.table {
+  position: relative;
+  width: 100%;
+  max-width: 100%;
+  overflow-y: hidden;
+  overflow-x: auto;
+  height: 100%;
+  min-height: 3rem;
+  font-family: var(--sapFontFamily);
+  font-size: var(--sapFontSize);
+  font-weight: normal;
+  background-color: var(--sapList_Background);
+
+  /*  bottom border*/
+  &::after {
+    content: '';
+    position: absolute;
+    inset-block-end: 0;
+    inset-inline-start: 0;
+    height: 1px;
+    width: 100%;
+    background: var(--sapList_TableFooterBorder);
+  }
+}
+
+.tableContainerWithScrollBar {
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
could be replaced with inset: 0?

Reply:
I kept it like this as it shouldn't make a difference for different writing-modes, but this way we're save if these styles are overwritten somewhere else.